website/content/api-docs/system/init.mdx

```diff
@@ -60,8 +60,6 @@ available when using Vault HSM.
   `secret_shares`. If using Vault HSM with auto-unsealing, this value must be
   the same as `secret_shares`.
 
-Additionally, the following options are only supported on Vault Pro/Enterprise:
```

Comment:
Instead of removing this altogether, how about something like "... only supported when using Auto Unseal"?  

There's also a bit higher up where we say "only available when using Vault HSM" that should also be adjusted similarly.

Reply:
@greut Just wanted to drop a note, there is one change that was suggested that is pending, which is to reword the `only available when using Vault HSM` bit under the `Start Initialization` section.